testing/matrix-sdk-ffi-macros/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+description = "Helper macros to write FFI bindings"
+edition = "2021"
+homepage = "https://github.com/matrix-org/matrix-rust-sdk"
+keywords = ["matrix", "chat", "messaging", "ruma"]
+license = "Apache-2.0"
+name = "matrix-sdk-ffi-macros"
+readme = "README.md"
+repository = "https://github.com/matrix-org/matrix-rust-sdk"
+rust-version = { workspace = true }
+version = "0.7.0"
```

Comment:
authot is missing :-).

Reply:
We don't usually set it, or we set it to Damir. Would need to do something more general for all the crates we have :)

Reply:
like setting all to "The Rust Matrix SDK developers" or something like this